templates/advancement/size-config.hbs

```diff
@@ -0,0 +1,10 @@
+<form autocomplete="off">
+  {{> "dnd5e.advancement-controls"}}
+
+  <div class="form-group">
+    <label>{{localize "DND5E.AdvancementHint"}}</label>
+    <textarea name="configuration.hint">{{configuration.hint}}</textarea>
+  </div>
```

Comment:
I saw in your screenshot that this looked quite cramped. It would be nicer to have the textarea able to fill the full width I think.

Reply:
<img width="408" alt="Screenshot 2023-11-06 at 10 19 12" src="https://github.com/foundryvtt/dnd5e/assets/19979839/eeb552fd-f3e5-4531-b62c-2d294007a0c8">